src/components/VLicense/VLicenseElements.vue

```diff
@@ -56,11 +56,13 @@ export default defineComponent({
     )
 
     const isSmall = computed(() => props.size === 'small')
+    const isMobile = computed(() => !isMinScreen('md').value)
```

Comment:
Is the intention here to get the small screen, no matter what user agent the request sends? Then it's better to use `!uiStore.isDesktopLayout`. If you need the mobile UA specifically, then you can use 
https://github.com/WordPress/openverse-frontend/blob/4bd35dc8f7e768c1e1f39b83c8fa30f56c9fe4a8/src/stores/ui.ts#LL59C6-L59C6

This way the component is rendered correctly on the server side.

Reply:
Wouldn't relying on UA for the mobile version would the storybook visual regression tests? That's why I was relying on the breakpoint. I'll update the code to use `uiStore`.

Reply:
I was thrown off by `isMobile` and thought for a bit that it's about the UA specifically. The breakpoints in the UI store work no matter which UA the request has. But, in addition to that, you might need some mobile features - in which case you can use the `isMobile` in the UI store.

Reply:
Updated in f42aa9c.